front/lib/api/assistant/configuration.ts

```diff
@@ -47,10 +47,27 @@ export async function getAgentConfiguration(
     return await getGlobalAgent(auth, agentId);
   }
 
+  const latestVersion = await AgentConfiguration.max<
```

Comment:
We can surely do it in one query no?

Reply:
With sequelize I couldn't find how, will look again

Reply:
found a solution

Reply:
fixed